testing/gerbv_example/slots-milldrill/expected/milldrill.ngc

```diff
@@ -577,558 +577,303 @@ G1 X3.95183 Y-2.39173
-G1 Z-1.60000
-G2  X4.46063 Y-2.85000 I-0.00906 J0.00000
+G1 Z0.02963
+G2 X4.46063 Y-2.85000 Z-0.00000 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.02963 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.05926 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.08889 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.11852 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.14815 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.17778 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.20741 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.23704 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.26667 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.29630 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.32593 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.35556 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.38519 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.41481 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.44444 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.47407 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.50370 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.53333 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.56296 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.59259 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.62222 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.65185 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.68148 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.71111 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.74074 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.77037 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.80000 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.82963 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.85926 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.88889 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.91852 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.94815 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-0.97778 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.00741 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.03704 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.06667 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.09630 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.12593 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.15556 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.18519 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.21481 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.24444 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.27407 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.30370 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.33333 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.36296 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.39259 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.42222 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.45185 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.48148 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.51111 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.54074 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.57037 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.60000 I-0.00906 J0.00000
+G2 X4.46063 Y-2.85000 Z-1.60000 I-0.00906 J0.00000
```

Comment:
We don't need the Z on the second time around.  The downside is that someone looking at it might think that the line was repeated on accident and remove one of them.  But making a special case for this makes the code longer and more ugly so I think that it's okay to leave it like this.

Reply:
Yeah. I thought so too and was too lazy for the special case. :-)

But I implemented something and it doesn't look that bad:
```
          if (!slot) {
            // Just drill a full-circle.
            of << arc_gcode
               << " X" << start_targetx * cfactor
               << " Y" << start_targety * cfactor;
            // Drop superflous Z from the bottom hole to indicate that this line is not there by accident
            if(stepcount != current_step) {
               of << " Z" << z * cfactor;
            }
            of << " I" << (start_x-start_targetx) * cfactor
               << " J" << (start_y-start_targety) * cfactor << "\n";
          } else {
```

Reply:
OK. Committed. Will update the tests once they have failed and I can get the patch.

I will see about the slots, but I'm not making any promises as I don't know if I will have the time.

And I need to fix the slot problem on my side before that. I hope it's just the outdated gerbv version. I don't want to update libboost on my computer as it has too many dependencies...

Reply:
I think that it's just gerbv that needs building because only the slots looked wrong.  You can see how gerbv is installed here:

https://github.com/pcb2gcode/pcb2gcode/blob/master/.travis.yml#L79

If you're unable to also support the slots, that's okay, too.  I might get around to it.